README.md

```diff
@@ -1,24 +1,53 @@
-[![develop build coverage](https://coveralls.io/repos/github/dahlbyk/posh-git/badge.svg?branch=develop)](https://coveralls.io/github/dahlbyk/posh-git?branch=develop)
-
-[![Join the chat at https://gitter.im/dahlbyk/posh-git](https://badges.gitter.im/dahlbyk/posh-git.svg)](https://gitter.im/dahlbyk/posh-git?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![posh-git on Chocolatey](https://img.shields.io/chocolatey/dt/poshgit.svg)](https://chocolatey.org/packages/poshgit/)
+## Overview
 
 posh-git is a PowerShell module that integrates Git and PowerShell by providing Git status summary information that can be displayed in the PowerShell prompt, e.g.:
+
 ```powershell
 C:\Users\Keith\GitHub\posh-git [master ≡ +0 ~1 -0 !]>
 ```
+
 posh-git also provides tab completion support for common git commands, branch names, paths and more.
 For example, with posh-git, PowerShell can tab complete git commands like `checkout` by typing `git ch` and pressing the <kbd>tab</kbd> key.
 That will tab complete to `git checkout` and if you keep pressing <kbd>tab</kbd>, it will cycle through other command matches such as `cherry` and `cherry-pick`.
 You can also tab complete remote names and branch names e.g.: `git pull or<tab> ma<tab>` tab completes to `git pull origin master`.
 
-## Versions
+### Build status
+
+| AppVeyor (Windows) | Travis (Linux) | Travis (macOS) | Code Coverage Status |
+|--------------------|----------------|----------------|----------------------|
+| [![master build status][av-master-img]][av-master-site] | | | [![master build coverage][cc-master-img]][cc-master-site] |
+| [![develop build status][av-develop-img]][av-develop-site] | [![develop pscore build status][tv-develop-img]][tv-develop-site] | PLACEHOLDER | [![develop build coverage][cc-develop-img]][cc-develop-site] |
+
+## Supported Versions
+
+Two versions of posh-git are available to provide support for Windows PowerShell v2 up through PowerShell Core v6.
+
+### posh-git v0.x
+
+Version 0.x of posh-git continues to support Windows PowerShell versions 2, 3 and 4.
+And if you don't need ANSI escape sequence support, v0.x can be used on Windows PowerShell v5.x.
+See that version's [README][!!!PATH-TO-0x-README-HERE!!!] for installation instructions.
+
+### posh-git v1.x
+
+Version 1.x of posh-git is targeted specifically at Windows PowerShell 5.x and (cross-platform)
+PowerShell Core 6.x, both of which support writing prompt strings using [ANSI escape sequences][ansi-esc-code].
+On Windows, support for [Console Virtual Terminal Sequences][console-vt-seq] was added in Windows 10 version 1511.
+Consequently this version of posh-git introduces BREAKING changes with 0.x which including dropping support
+for Windows PowerShell version 2, 3 and 4 since these versions of PowerShell do not run on platforms that
```

Comment:
Is it actually true that PowerShell pre-5 doesn't support prompts with ANSI escape sequences? I haven't tried, but I see no reason it wouldn't work with a proper host. Furthermore, 1.0 _should_ still work on hosts without ANSI support.

It feels like you're blaming ANSI for the breaking changes, when they're mostly motivated (I think?) by improving the user experience (settings refactor) and making our lives easier dev-wise (newer languages features).

Reply:
Well it is v5 (or v5.1) that adds the `$host.UI.SupportsVirtualTerminal` property we use to set the AnsiConsole setting.  But I suppose someone could be using ConEmu or a shell like that.  That said, posh-git 1.x won't run on anything less than v5 because of our use of PS classes.

RE blaming ANSI - I'm fine with rewording that to shift the emphasis to the settings restructuring.

Reply:
Reworded.